04/app.js

```diff
@@ -1,19 +1,34 @@
 const divElements = document.querySelectorAll('div');
 if(divElements) {
     divElements.forEach(function(item) {
-        item.addEventListener('click', newClass )
-    })
+        item.addEventListener('click', addNewClass )
+    });
 }
 
-document.body.addEventListener('click', removeClass)
+document.body.addEventListener('click', removeClickedFromDivs)
 
-function newClass() {
+function addNewClass(e) {
+    //pokazanie rónicy pomiędzy e.target a e.currentTarget
+    // console.log('kliknięto na: ') 
+    // console.log(e.target)
+    // console.log('gdzie jest callback: ')
+    // console.log(e.currentTarget);
+    const divElement = e.currentTarget;
+    const timeout = divElement.dataset.time
 
-}
-
-function removeClass() {
+    setTimeout(function() {
+        divElements.classList.add('clicked')
```

Comment:
Zobacz, że w konsoli jest błąd, który sugeruje, że coś jest nie tak:

![image](https://user-images.githubusercontent.com/55095257/210328471-234fa356-55d6-476d-8ce1-61402b1a6970.png)

Musisz to każdego elementu z osobna dodać klasę a nie do wszystkich (tutaj jest błąd).